honeybee_3dm/grid.py

```diff
@@ -0,0 +1,77 @@
+"""Import Rhino surfaces as Honeybee grid objects from a Rhino3DM file."""
+import warnings
+# The Rhino3dm library provides the ability to access content of a Rhino3dm
+# file from outside of Rhino
+import rhino3dm
+
+# Importing Honeybee dependencies
+from honeybee_radiance.sensorgrid import SensorGrid
+from honeybee.typing import clean_and_id_string, clean_string
+from ladybug_geometry.geometry3d import Face3D, Mesh3D
+
+# Importing dependencies from Honeybee-3dm package
+from .togeometry import brep2d_to_face3d, mesh_to_face3d, mesh_to_mesh3d
+from .helper import filter_objects_by_layer
+
+
+# TODO: expose an option to change the target layer name
+# TODO: Grid size and offset will have to provided using the config file
```

Comment:
Looks like you have already addressed this TODO.

Reply:
I merely exposed on the method. There's no way for a user to provide these inputs as of now so I kept these here.

Reply:
I see. Good point. I might suggest to create an issue for this or add the TODO to the related module.

Reply:
Created [here](https://github.com/ladybug-tools/honeybee-3dm/issues/5)